Swiftfin.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,9 +2,12 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:JellyfinPlayer.xcodeproj">
+      location = "group:/Users/epippin/Developer/GitHub/SwiftFin/Swiftfin.xcodeproj">
```

Comment:
Should this be a relative path, and not one from your dev machine?

Reply:
oh woops, good catch